src/main/java/carpet/script/api/Auxiliary.java

```diff
@@ -276,6 +276,14 @@ public static void apply(Expression expression)
             ));
         });
 
+        expression.addContextFunction("item_name_text", -1, (c, t, lv)->{
```

Comment:
`item_name_text` sounds strange, I'd suggest something more like `item_display_name` (or maybe `item_name` if there's nothing like that already, though not sure), because it being a text isn't something worth noting in the method name IMO (given it's interchangeable with a string).

Reply:
Actually wasn't this already `item_display_name` before?